lib/logstash/outputs/elasticsearch.rb

```diff
@@ -103,6 +103,11 @@ class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
   # not also set this field. That will raise an error at startup
   config :path, :validate => :string
 
+  # Pass a set of key value pairs as the URL query string. This query string is added
+  # to every host listed in the 'hosts' configuration. If this list contains urls which
+  # already contain a query string, this one will be appended to the existing one.
```

Comment:
oh, I meant, if you paste a url in `hosts` like `hosts => [ "http://localhost?random=thing" ]`, the specified query parameters are added to `random=thing`